ligand/views.py

```diff
@@ -625,9 +676,10 @@ def get_context_data(self, **kwargs):
         self.request.session.modified = True
         rd = list()
         for i in datum.split(','):
-            ligand = Ligand.objects.filter(id=int(i))
+            ligand = Ligand.objects.filter(id=i)
             ligand = ligand.get()
-            links = LigandVendorLink.objects.filter(lp=ligand.properities_id).prefetch_related('lp','vendor')
+            links = LigandVendorLink.objects.filter(
+                lp=ligand.properities_id).prefetch_related('lp', 'vendor')
             for x in links:
                 if x.vendor.name not in ['ZINC', 'ChEMBL', 'BindingDB', 'SureChEMBL', 'eMolecules', 'MolPort', 'PubChem']:
                     temp = dict()
```

Comment:
`vendor = LigandVendorLink.objects.filter(id=x.vendor_id)`
instead of:
`vendor = LigandVendors.objects.filter(id=x.vendor_id)`

Reply:
Done